services/mac_manager/src/test/java/com/futurewei/alcor/macmanager/swagger/SwaggerConfig.java

```diff
@@ -14,7 +14,7 @@
         limitations under the License.
 */
 
-package com.futurewei.alcor.macmanager.config;
+package com.futurewei.alcor.macmanager.swagger;
```

Comment:
No need to touch swagger file as they are commonly used by other services as well for Swagger UI generation.

Reply:
Yes, I will not touch the file.

Reply:
Please revert this change.